### PR TITLE
tests: move MockView directly into urls module

### PR DESCRIPTION
**Pull request by blueyed**
It is only uses to create the URLs, for testing the middleware against
it.

### CODE REVIEW

File: pinax/stripe/tests/urls.py [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
 from django.conf.urls import url
-from .mock_views import MockView
 from ..urls import urlpatterns
 
-urlpatterns += [
 
-    url(r"^admin/", admin.site.urls),
+class FakeViewForUrl(object):
+    def __call__(self):
+        raise Exception('Should not get called.')
```
Comment (paltman) [inline pinax/stripe/tests/urls.py]: can you fix the single-quotes so I can merge this?

**Comment by blueyed** [comment]
Sorry, but I do not care about cleaning up other peoples stuff anymore.

**Comment by blueyed** [comment]
As a project maintainer you might want to check out https://github.com/github/hub for easily cherry-picking and amending some commit.

I am sorry if this might offend you, I am just tired of this.

**Comment by blueyed** [comment]
updated..

**Comment by blueyed** [comment]
btw: you should maybe use the same flags on Travis/CircleCI (with e.g. adding "travis" for Travis builds), or just get rid of Travis soon, like you are probably trying to do.

**Comment by paltman** [comment]
Yeah, it's a bit messy right now.  Travis is no longer active on `master`.  Sorry about the noise.

**Pull request merged**